src/controllers/profile.js

```diff
@@ -1,18 +1,49 @@
 import { dirname, resolve } from 'node:path';
 import { fileURLToPath } from 'node:url';
 
-import express from 'express';
 import ArsenalParser from '@wfcd/arsenal-parser';
+import ProfileParser from '@wfcd/profile-parser';
+import express from 'express';
 import flatCache from 'flat-cache';
 
-import { noResult, cache } from '../lib/utilities.js';
+import settings from '../lib/settings.js';
+import { cache, noResult } from '../lib/utilities.js';
 
 const router = express.Router({ strict: true });
 
-const WF_ARSENAL_ID = 'ud1zj704c0eb1s553jbkayvqxjft97';
-const WF_ARSENAL_API = 'https://content.warframe.com/dynamic/twitch/getActiveLoadout.php';
-let token;
+const get = async (username) => {
+  const profileUrl = `${settings.wfApi.profile}?n=${encodeURIComponent(username)}`;
+  const data = await fetch(profileUrl, { headers: { 'User-Agent': process.env.USER_AGENT || 'Node.js Fetch' } });
+
+  if (data.status !== 200) return undefined;
+
+  return data.json();
+};
+
+router.get('/:username/?', cache('1 minute'), async (req, res) => {
```

Comment:
could you consider caching these for like.... an hour?  we can document, but anyone trying to fetch it "live" is doing overkill...

Reply:
tis done